CHANGELOG.md

```diff
@@ -1,3 +1,34 @@
+# v0.10.0 (*08.08.2024*)
+
+## Major Changes since v0.9.0
+* Automatic network generation from high-level specifications.
+* Units at the user interface including scaling and conversion.
+* Morphologies are now loaded through a unified interface, bundling morphology, meta data, and segment tree.
+
+## Internal Updates
+
+* Documentation overhaul
+* Performance improvements
+  - Label resolution uses hashes instead of strings
+  - Spike delivery is now up to 30% faster leading 
+  - Load balancing
+* Fixes
+  - Better support for source builds on MacOS (aarch64)
+  - `modcc` no longer allows internal variables (`v`, `celsius`, ...) as `ASSIGNED`
+  - Better support for raw (C++) mechanisms
+* MC cells are now cable cells 
```

Comment:
what is MC cells?

Reply:
Stands (stood?) for Multi-Compartment as in `mc_cell_group`. However, all text and interface calls them `cable`. Thus we removed this inconsistency by renaming the code. It's purely cosmetic.